pkgs/applications/networking/znc/modules.nix

```diff
@@ -94,6 +94,7 @@ in {
       description = "ZNC FiSH module";
       homepage = https://github.com/dctrwatson/znc-fish;
       maintainers = [ stdenv.lib.maintainers.offline ];
+      broken = true;
```

Comment:
znc backported fix https://github.com/NixOS/nixpkgs/commit/cc25b7a7bd17624daf4f7f60a99f8dc8266215c0

Reply:
Thanks, removed this broken-marking :)